docs/designers-developers/developers/tutorials/devenv/readme.md

```diff
@@ -1,14 +1,65 @@
 # Development Environment
 
-The development environment setup guide is for setting up your local development environment for JavaScript development; creating plugins and tools for extending WordPress and the block editor.
+This guide is for setting up your local environment for JavaScript development for creating plugins and tools to extend WordPress and the block editor.
 
-A development environment is a catch-all term for the setup on your computer to work. The three main pieces needed for our development environment are:
+A development environment is a catch-all term for what you need setup on your computer to work. The three main pieces needed for our development environment are:
 
 1. Node/NPM Development Tools
 2. WordPress Development Site
 3. Code Editor
 
-## Development Tools
+## Quickstart
+
+For experienced developers, here is a summary of the guide, see each section for more details and explanations.
+
+**1. Install Node development tools**
+
+Download and install [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm)
+
+```
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+```
+
+Quit and restart terminal
+```
+nvm install --lts
+```
+
+**2. WordPress Development Site**
+
+First download, install, and start [Docker Desktop](https://www.docker.com/products/docker-desktop) follows instructions on their site for your OS.
+
+- Install WordPress environment tool
+
+```
+npm -g install @wordpress/env
+```
+
+Open a text editor and create a `.wp-env.json` file with the following content:
+
+```json
+{
+	"core": null
+}
+```
+
+This is a minimal file, you can also specify various features and plugins, see documentation for full details. The `"core": null` parameter indicates no version specified, so it defaults to the latest version.
+
+ Start the environment:
+
+ ```
+ wp-env start
+ ```
+
+You will have a full WordPress site installed, navigate to: http://localhost:8888/ the defualt user is "admin" and default password is "password"
+
```

Comment:
In which folder is the localhost web site located?

Reply:
It is in Docker, not a folder.

Reply:
Ahhh....

` wp-env start`

You will have a full WordPress site installed, navigate to: http://localhost:8888/ the defualt user is "admin" and default password is "password"

Another way to open the local WordPress site after it has been started up is through Docker. 
Open Docker and go to the Dashboard. 

![Docker-Dashboard](https://user-images.githubusercontent.com/5323259/92227783-13f51780-eea7-11ea-97fe-7730bf166fe0.png)

Click where it says RUNNING. A new screen opens and it will show the containers (what is a container?). 

![Docker containers](https://user-images.githubusercontent.com/5323259/92227672-e4dea600-eea6-11ea-9a9a-779c0ccfc3b6.png) 

Hovering the cursor over one of the containers shows various options. Among them "OPEN IN BROWSER"
Clicking it will then open the local site in the port mentioned.